manager-bundle/bin/contao-script-handler

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+use Contao\ManagerBundle\Console\ContaoApplication;
+use Contao\ManagerBundle\HttpKernel\ContaoKernel;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Filesystem\Filesystem;
+
+set_time_limit(0);
+@ini_set('zlib.output_compression', '0');
+
+if (file_exists(__DIR__.'/../autoload.php')) {
+    $projectDir = \dirname(__DIR__, 2);
+} elseif (file_exists(__DIR__.'/../../../../autoload.php')) {
+    $projectDir = \dirname(__DIR__, 5);
+} elseif (false !== ($cwd = getcwd()) && file_exists($cwd.'/vendor/autoload.php')) {
+    $projectDir = $cwd;
+} else {
+    $projectDir = \dirname(__DIR__, 4);
+}
+
+require $projectDir.'/vendor/autoload.php';
+
+$kernel = ContaoKernel::fromInput($projectDir, new ArrayInput(['--env' => 'prod']));
```

Comment:
I think we should not use the Contao application here, but create our own application that only has one command. Similar to the `console-api` (though I hope we can do that without a child class of `Application`)

Reply:
Why that? Don't we explicitly want the Contao application with its extensions so that we can inject things like the web-dir or other services?

Reply:
We want the Contao application and its configuration for the final command (cache clear, symlinks etc). But _executing_ these commands does not need the kernel or the configuration imho.